src/client/action/createRow.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class CreateRow extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
It seems to mismatching for type of `seq`. Expected `number` but got `string` if we create or update the row.

```js
export const rowPropType = PropTypes.shape({
  seq: PropTypes.number,
  status: PropTypes.string,
  category: PropTypes.string,
  title: PropTypes.string,
  owner: PropTypes.string,
  priority: PropTypes.string,
```